### ISSUE TITLE
Mic icon not disappearing in the browser tab. after record is stopped

### ISSUE THREAD

**Issue by naveenbharadwaj19** [llfbandit/record#314]
**Package version**
5.0.4

**Environment**
 - OS: windows
 - Browser : Brave,Chrome

![2024-05-17 13_40_11-Sarathy - Brave](https://github.com/llfbandit/record/assets/63662884/a902d6f1-1fac-416f-9f39-86031f967154)

**Describe the bug**
The problem is that the red dot in the tab doesn't go away even after the recording is stopped. I checked if something is being recorded by using `record.isRecording()`, and I'm getting `false`. https://github.com/llfbandit/record/issues/260. Still facing same issue

**code snippet**

```dart
final record = AudioRecorder();
_stream = await record.startStream(const RecordConfig(
          encoder: AudioEncoder.pcm16bits,
          sampleRate: 22000,
          numChannels: 2,
          bitRate: 128000));
          
  Future<void> stopRecording() async {
    var recordOutput = await record.stop();
    _isRecording = false;
    print("record output : $recordOutput");
    notifyListeners();
  }

**Comment by llfbandit** [comment]
This should be fixed in next record_web: 1.1.0. I didn't reproduced it with the example.
Feedback welcome.

**Comment by llfbandit** [comment]
`record_web` 1.1.0 has been released.

**Comment by naveenbharadwaj19** [comment]
Thanks, it has been resolved in version 5.1.0.
